chatbot-ui/hooks/useChatbot.test.ts

```diff
@@ -12,7 +12,7 @@ describe('useChatbot', () => {
     const { messages } = result.current;
     expect(messages).toHaveLength(1);
     expect(messages[0].content).toBe(
-      'Hi, what would you like to know about the latest publications on EES?',
+      'Hi, what would you like to know about on Explore Education Statistics?',
```

Comment:
Minor typos here I think? Could remove the " on ". Same for lines below :)

Reply:
Removed the 'on' now, thanks. 🙂  Not sure I can spot the typo?